system_metrics_collector/src/topic_statistics_collector/parameter_utils.hpp

```diff
@@ -82,6 +82,11 @@ static rcl_interfaces::msg::ParameterDescriptor BuildTopicParameterDescriptor(
  *
  * @throws std::invalid_argument if the parameter value is empty
  */
+static void CheckEmptyStringParam(const std::string & name, const std::string & value)
```

Comment:
This method is exactly the same as the one below. Could we reuse that one (rename it if you want), instead of adding a duplicate?